server/resolvers/article/queries.js

```diff
@@ -1,11 +1,14 @@
+import axios from 'axios'
+import { SEARCH_SERVICE_ENDPOINT } from '../../config'
+
 const articleQueries = {
-  articles: (_, { search }) => {
-    return [
+  articles: async (_, { search }) => {
```

Comment:
When the db endpoint is available this should be the only thing in this file right?

Reply:
Exactly, I'm actually already switching from the mocked data to the actual request to the DB endpoint.